geomstats/learning/k_means.py

```diff
@@ -0,0 +1,87 @@
+import random
+
+import geomstats.backend as gs
+from geomstats.learning._template import TemplateTransformer
+
+
+class KMeans(TemplateTransformer):
+
+    def __init__(self, n_clusters, metric, init="random",
```

Comment:
single quotes ' instead of "
--> Just because we need to decide on one for code uniformization, and we had chosen ' before.

Reply:
let's put the metric as the first parameter after `self`, as in:
https://github.com/geomstats/geomstats/blob/master/geomstats/learning/pca.py#L100
since it's the only parameter that differs from the scikit-function.

Also, let's set `n_clusters=8` by default, as in scikit-learn?
https://github.com/scikit-learn/scikit-learn/blob/b194674c4/sklearn/cluster/_kmeans.py#L793